charts/vector/templates/haproxy/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.haproxy.enabled }}
+{{- if and .Values.haproxy.enabled (not .Values.existingConfigMap) }}
```

Comment:
Does `not` on an empty string return `true`?

Reply:
👍  empty is ~= false when evaluating templates